README.rst

```diff
@@ -1,21 +1,190 @@
+
+  # Using your favorite editor, edit the code to make your change.
+  vim ...
+
+  # Temporary until https://github.com/openedx/xblock-utils/issues/199 is resolved
+  mkdir var
+  touch var/workbench.log
+
+  # Run your new tests
+  pytest ./path/to/new/tests
+
+  # Run all the tests and quality checks
+  make test
+
+  # Commit all your changes
+  git commit ...
+  git push
+
+  # Open a PR and ask for review.
+
+Deploying
+=========
+
+This component is automatically deployed to PyPI whenever new GitHub releases are made.
+
+To deploy this library.
+
+#. Update the version in ``xblockutils/__init__.py`` based on semantic versioning.
+
+#. Create a new GitHub release with a tag matching the version.
+
+#. Automation should build and deploy the version to PyPI
+
+Getting Help
+************
+
+Documentation
+=============
+
+Start by going through `the documentation`_.  If you need more help see below.
+
+.. _the documentation: https://docs.openedx.org/projects/xblock-utils
+
+More Help
+=========
+
+If you're having trouble, we have discussion forums at
+https://discuss.openedx.org where you can connect with others in the
+community.
+
+Our real-time conversations are on Slack. You can request a `Slack
+invitation`_, then join our `community Slack workspace`_.
+
+For anything non-trivial, the best path is to open an issue in this
+repository with as many details about the issue you are facing as you
+can provide.
+
+https://github.com/openedx/xblock-utils/issues
+
+For more information about these options, see the `Getting Help`_ page.
+
+.. _Slack invitation: https://openedx.org/slack
+.. _community Slack workspace: https://openedx.slack.com/
+.. _Getting Help: https://openedx.org/getting-help
+
+License
+*******
+
+The code in this repository is licensed under the AGPLv3 unless
+otherwise noted.
+
+Please see `LICENSE <LICENSE>`_ for details.
+
+Contributing
+************
+
+Contributions are very welcome.
+Please read `How To Contribute <https://openedx.org/r/how-to-contribute>`_ for details.
+
+This project is currently accepting all types of contributions, bug fixes,
+security fixes, maintenance work, or new features.  However, please make sure
+to have a discussion about your new feature idea with the maintainers prior to
+beginning development to maximize the chances of your change being accepted.
+You can start a conversation by creating a new issue on this repo summarizing
+your idea.
+
+The Open edX Code of Conduct
+****************************
+
+All community members are expected to follow the `Open edX Code of Conduct`_.
+
+.. _Open edX Code of Conduct: https://openedx.org/code-of-conduct/
+
+People
+******
+
+The assigned maintainers for this component and other project details may be
+found in `Backstage`_. Backstage pulls this data from the ``catalog-info.yaml``
+file in this repo.
+
+.. _Backstage: https://open-edx-backstage.herokuapp.com/catalog/default/component/xblock-utils
```

Comment:
Why don't we use backstage.openedx.org as the canonical domain of Backstage? It would prevent this link from breaking if we ever move off of Heroku.